flexget/manager.py

```diff
@@ -1052,6 +1037,23 @@ def _shutdown(self) -> None:
         global manager
         manager = None
 
+    def tasks_extractor(self, tasks):
```

Comment:
Not sure about the name... maybe `matching_tasks`? Also, let's add some type hints here.

Other questions I have, which I haven't decided my full stance:
- Should it raise an exception rather than logging one?
- Should it act on a single string rather than a list?

I'm leaning towards yes on both of those, and let the caller be responsible for looping and logging.